egs/chime6/asr1/local/install_pb_chime5.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Installs pb_chime5
+# miniconda should be installed in $HOME/miniconda3/ 
+
```

Comment:
Can we reuse the python venv from espnet installation here to install GSS instead of reinstalling miniconda. @sw005320 What do you think?

Reply:
I think this is a good idea.

Reply:
Thanks for this suggestion!
I'll try to do it.